eng/Signing.props

```diff
@@ -58,16 +58,6 @@
     <FileSignInfo Include="Mono.Cecil.Rocks.dll" CertificateName="3PartySHA2" />
   </ItemGroup>
 
-  <!--
-    Ensure that we sign the AZL3 RPM with the LinuxSignMariner key.
-    This package name has the version in it, so we need to use a wildcard to discover the right name.
-  -->
-  <ItemGroup>
-    <AzureLinuxRPM Include="$(ArtifactsPackagesDir)**/*-azl.*-*.rpm" />
```

Comment:
We need to keep this one for the prereqs package

Reply:
I also moved that to the arcade PR.

https://github.com/dotnet/arcade/pull/15684/files#diff-41603720a0113db0f6fbe68a4bf6d873a5ba4affb2f7f6238ac19eed4c97b008R99-R101